docs/site/WEBSTORM.md

```diff
@@ -0,0 +1,85 @@
+# Developing with Webstorm
+
+There is currently an issue opened with Webstorm :
+[WEB-34416](https://youtrack.jetbrains.com/issue/WEB-34416)
+
+It should be fixed in `2018.3`, WebStorm is indexing symlinks create by Lerna
+inside `node_modules`, so it takes ages. As a workaround you need to exclude
+every `node_modules` directory or you can use [VisualStudio Code](./VSCODE.md).
+
+We recommend our contributors to use
+[WebStorm](https://www.jetbrains.com/webstorm/) with the following extensions
+installed:
```

Comment:
As I am reading this sentence, it's saying that the LoopBack project recommends people contributing code to use WebStorm. I am rather reluctant to recommend WebStorm because most of the team is using VS Code now.  I expect this will change in the future as we recruit more maintainers from our community, but we are not there yet.

So, I am proposing to use the following phrasing:

```markdown
For contributors using [WebStorm](https://www.jetbrains.com/webstorm/) as their preferred 
development environment, we recommend the following extensions to be installed:
```